velodyne_pointcloud/src/lib/rawdata.cc

```diff
@@ -120,6 +124,35 @@ namespace velodyne_rawdata
    return 0;
   }
 
+
+  std::vector<std::vector<double>> RawData::getVLP32TimingOffsets() {
+    // timing table calculation, from velodyne user manual
+
+    // 12 firings cycles in a data package, 32 lasers:
+    std::vector<std::vector<double>> timing_offsets(12, std::vector<double>(32, 0.));
+
+    // Time constants
+    const double full_firing_cycle = VLP_32_SPEC.firing_seq_duration * 1e-6;
+    const double single_firing = VLP_32_SPEC.firing_duration * 1e-6;
+    const bool dual_mode = false;
```

Comment:
We can in theory set the VLP32 to use dual mode. We should support this (not hard code to false but take from config)

Reply:
The value is not in the config.